pkg/types/openstack/validation/machinepool_test.go

```diff
@@ -71,6 +79,21 @@ func TestValidateMachinePool(t *testing.T) {
 				exactlyNErrors(1),
 			),
 		},
+		{
+			"with availability zone and valid root volume",
+			testMachinePool(withAvailabilityZones([]string{"az0", "az1", "az2"}), withRootVolume(openstack.RootVolume{Zones: []string{"az0", "az1", "az2"}, Size: 100})),
+			"default",
+			check(noError),
+		},
+		{
+			"with availability zone and invalid root volume",
+			testMachinePool(withAvailabilityZones([]string{"az0", "az1", "az2"}), withRootVolume(openstack.RootVolume{Size: 0})),
```

Comment:
Why isn't this also triggering an error with the volume size?

Reply:
The Size validation exists but in the other file where "dynamic" validations are. I think this is fine as it is now?
Do you see anything I should change?

Reply:
https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/openstack/validation/machinepool.go#L44-L56